systems/data-plan/package/pb/package.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+package ukama.packages.v1;
+option go_package = "pb/gen";
+
+service PackagesService {
+    rpc Get (GetPackagesRequest) returns (GetPackagesResponse) {}
```

Comment:
Get returns only one package (or none), so it should be `GetPackageRequest` instead of GetPackage**s**Request and the response should be `GetPackageResponse` instead of GetPackage**s**Response